CHANGELOG.md

```diff
@@ -216,6 +216,9 @@ Adding a new version? You'll need three changes:
   those names will be propagated to one or many instances of aforementioned tag.
   [#6759](https://github.com/Kong/kubernetes-ingress-controller/pull/6759)
   [#6780](https://github.com/Kong/kubernetes-ingress-controller/pull/6780)
+- Added the flag `--secret-label-for-caching` to set KIC to only cache `Secrets` with a specified label and value set
```

Comment:
I think that some users might be confused about the mention of "caching" here. How about naming this simply 
```suggestion
- Added the flag `--secret-label-selector` to set the label selector for `Secrets` to ingest. 
```

?

Reply:
This is indeed simpler, let me make some changes.